translations/en-us.yml

```diff
@@ -100,6 +100,9 @@ general:
         preprints: Preprints
         registries: Registries
     other: Other
+    structured_data:
+        json_ld_retrieval_error: 'Error retrieving JSON-LD object for Google Structured Data.'
+        environment_error: 'Error identifying environment.'
```

Comment:
`environment_error` does not seem to be used. And if we don't throw an exception, `json_le_retrieval_error` would also not be used.

Reply:
Ok, environment_error is now removed. json_ld_retrieval_error will be repurposed as discussed to include the error object for our records to Sentry.